### ISSUE TITLE
Ability to search in Suceedeed jobs

### ISSUE THREAD

**Issue by franklbt** [HangfireIO/Hangfire#2287]
It would be usefull to be able to search in the succeded jobs to find specific jobs and show their associated history.

Actually we need to navigate in all the page to try to find the job we want.

**Comment by mrjamiebowman** [comment]
Yes, I agree. I should be able to search and filter by job, job type, run time, etc. Especially, if I'm running a lot of jobs. The pagination is rather pointless.

**Comment by peterfortuin** [comment]
Searching in jobs would be awesome.
